target/linux/mvebu/image/cortex-a72.mk

```diff
@@ -0,0 +1,16 @@
+ifeq ($(SUBTARGET),cortexa72)
+
+define Device/armada-macchiatobin
+  KERNEL_NAME := Image
+  KERNEL := kernel-bin
+  DEVICE_TITLE := MACCHIATObin (SolidRun Armada 8040 Community Board)
+  DEVICE_PACKAGES := e2fsprogs ethtool mkf2fs kmod-fs-vfat kmod-usb2 kmod-usb3 kmod-usb-storage kmod-mmc mmc-utils pciutils
```

Comment:
Remove packages from community repository `mmc-utils`, `pciutils`. The USB drivers are built in the kernel, should be safe to remove them.